docs/07_how-to/14_ht-front-end-simple.mdx

```diff
@@ -0,0 +1,273 @@
+:::
+
+Here is an example declaration of a flyout:
+
+```ts
+<rapid-button @click=${x => x.leftFlyoutActive = true}>Open left positioned flyout</rapid-button>
+<rapid-button @click=${x => x.rightFlyoutActive = true}>Open right positioned flyout</rapid-button>
+<rapid-flyout :closed="${x => !x.rightFlyoutActive}" @closed=${() => console.log('right flyout closed')}>
+  <p>This paragraph will appear in the flyout</p>
+</rapid-flyout>
+<rapid-flyout :closed="${x => !x.leftFlyoutActive}" position="left" @closed=${() => console.log('left flyout closed')}>
+  <p>This paragraph will appear in the flyout</p>
+</rapid-flyout>
+```
+
+## Example application
+We have created an [example application](https://github.com/genesislcap/platform-howto/tree/4a140c118916b177c0dd5fd355a5d5ec36f7d33b/howto-uiexamples) to demonstrate the things you can set up on the front end of a Genesis application.
+
+There are multiple UIs, with links to the corresponding code.
+
+### The back end
+Even if you are a front-end specialist, it is important to have some familiarity with the back end. You need to know what resources you can connect to in order to retrieve data and interact, and in many cases you will need to know exactly what fields are available on a specific resource.
+
+In our case, the back end is purposely kept simple, with little functionality.  It contains:
+
+- static data tables for Entity and Client
+- a Trade table with 10 pre-loaded trades
+- Kafka topic subscription for FX Rates, to show ticking data in the front end
+- time-series data of inflation rates for use in charting
+
+:::info Take a look at the code
+The [**back-end code**](https://github.com/genesislcap/platform-howto/tree/4a140c118916b177c0dd5fd355a5d5ec36f7d33b/howto-uiexamples/server) of the example application has its own directory structure. 
+
+From there, you can drill down through the **uiexamples-app** to see the data files and the script files that define the back-end processes.
+:::
+
+:::tip Take a look at the app
+Check the [**readme file**](https://github.com/genesislcap/platform-howto/blob/4a140c118916b177c0dd5fd355a5d5ec36f7d33b/howto-uiexamples/server/README.md) before you try to run the application.
+
+The login is admin / genesis
+:::
+
+### The front end
+The principal aim of this example is to show you the code for the front end and enable you to run it and see the results.
+
+We recommend VSCode as your IDE for looking at the code if you are new to working with web code. This gives you full access to the Genesis plugin, which makes it easy to build and start the back end, and to start the front end.
+
+Alternatively, you can use a JetBrains IDE, such as IntelliJ.
+
+:::info Take a look at the code
+The [**back-end code**](https://github.com/genesislcap/platform-howto/tree/4a140c118916b177c0dd5fd355a5d5ec36f7d33b/howto-uiexamples/client/src) of the example application has its own directory structure. 
```

Comment:
Meant to say front end? we have link to back end in section above

points at a given commit not the master branch, this feels wrong. Also private repo.